aten/src/ATen/native/quantized/fake_quant_per_tensor_affine.cpp

```diff
@@ -108,7 +108,7 @@ int64_t _get_zero_point_from_tensor(
     bool is_forward) {
   float zero_point_fp = zero_point[0].item<float>();
   zero_point_fp = is_forward ? std::nearbyint(zero_point_fp) : zero_point_fp + 0.5f;
-  float zero_point_clamped = std::min(std::max(zero_point_fp, quant_min), quant_max);
+  float zero_point_clamped = std::min(std::max(zero_point_fp, float(quant_min)), float(quant_max));
```

Comment:
Why this change?

Reply:
Same as https://github.com/pytorch/pytorch/pull/51792#discussion_r607901624

Reply:
Please use `static_cast<float>(quant_min)` (and if I'm not mistaken, this change is already done in #https://github.com/pytorch/pytorch/pull/55638 )